src/pages/index.tsx

```diff
@@ -31,22 +36,22 @@ const HomepageButtons: FunctionComponent = () => {
 				_icons={{
 					right: 'codicon codicon-dashboard',
 				}}
-				_href="docs/get-started/first-steps"
+				_href={`docs/${version ? `${version}/` : ''}get-started/first-steps`}
 				_label={translate({
 					id: 'custom.get-started-button',
 				})}
 				_variant="primary"
 			></KolLinkButton>
 			<KolLinkButton
 				className="w-72"
-				_href="docs"
+				_href={`docs/${version ? `${version}` : ''}`}
 				_label={translate({
 					id: 'custom.documentation-button',
 				})}
 			></KolLinkButton>
 			<KolLinkButton
 				className="w-72"
-				_href={`/sample-react/#/handout/basic`}
+				_href={`${versionId}/sample-react/#/handout/basic`}
```

Comment:
Nicht ursprünglich von dir so umgesetzt, aber könntest du bitte allen drei Link hrefs ein `/` voranstellen? Dann funktionieren sie auch noch, wenn wir sie mal auf eine Unterseite der Dokumentation verschieben sollten.

```suggestion
				_href={`/${versionId}/sample-react/#/handout/basic`}
```